src/nls/root/strings.js

```diff
@@ -584,11 +584,9 @@ define(function (require, exports, module) {
                 RECENT_DOC: "Recent Documents"
             },
             CATEGORIES: {
-                CURRENT_DOC: "document",
-                RECENT_DOC: "document",
+                CURRENT_DOC: "current document",
```

Comment:
I want to make sure, are these going to be localized? if not, they shouldn't be defined here. If so, they should be capitalized properly.